cairo-sys-rs/Cargo.toml

```diff
@@ -10,6 +10,23 @@ keywords = ["cairo", "ffi", "gtk-rs", "gnome"]
 repository = "https://github.com/gtk-rs/cairo"
 build = "build.rs"
 
+[package.metadata.system-deps."cairo"]
+name = "cairo"
+version = "1.12"
+
+[package.metadata.system-deps."cairo".feature-versions]
+v1_14 = "1.14"
+v1_16 = "1.16"
+
+[package.metadata.system-deps."cairo-gobject"]
+name = "cairo"
```

Comment:
You need the actual lib name here, so  `cairo-gobject`, or you can just let it get it from the toml key name and remove this line.

Reply:
Oops. Thanks :)